DebugSwift/Sources/Features/App/App.Controller.swift

```diff
@@ -221,6 +233,14 @@ extension AppViewController {
                 actions.removeAll(where: { $0 == .console })
             }
 
+            if disabledActions.contains(.simulatePush) {
+                actions.removeAll(where: { $0 == .simulatePush })
+            }
+
+            if #available(iOS 13.0, *){} else {
```

Comment:
I think it works:

if !#available(iOS 13.0, *) {
    actions.removeAll { $0 == .simulatePush }
}

Reply:
you are right, my AI autocompleter I have integrated in my xcode did this weirdly hah